modules/apps/callbacks/types/callbacks.go

```diff
@@ -129,20 +135,32 @@ func GetCallbackData(
 	}
 
 	// get the gas limit from the callback data
-	executionGasLimit, commitGasLimit := computeExecAndCommitGasLimit(callbackData, remainingGas, maxGas)
+	executionGasLimit, commitGasLimit, err := computeExecAndCommitGasLimit(callbackData, remainingGas, maxGas)
+	if err != nil {
+		return CallbackData{}, true, err
+	}
```

Comment:
Can that be true, how are these lines not covered by tests? 🤔

Reply:
I think i was just testing lower level funcs, hopefully fixed now